must-gather/collection-scripts/gather_namespace_resources

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+# Expect base collection path as an argument
+BASE_COLLECTION_PATH=$1
+
+# Use PWD as base path if no argument is passed
+if [ "${BASE_COLLECTION_PATH}" = "" ]; then
+    BASE_COLLECTION_PATH=$(pwd)
+fi
+
+# Add general resources to list if necessary
+
+# Resource List
+resources=()
+# collect storagecluster resources
+
+# TODO: Re enable the collection of storagecluster via inspect command
+# resources+=(storageclusters)
+
+# collect OB/OBC resoureces
+resources+=(objectbucketclaims)
+resources+=(objectbuckets)
+resources+=(pvc)
+
+# collection path for OC commands
+mkdir -p ${BASE_COLLECTION_PATH}/oc_output/
+
+# Command List
+commands=()
+
+# collect oc output of OC get commands
+commands+=("clusterversion")
+commands+=("pods -owide")
+commands+=("nodes --show-labels")
+commands+=("pvc --all-namespaces")
+commands+=("pv")
```

Comment:
PersistentVolumes are cluster-scoped, not namespace-scoped. Please move these out to a new file called something like `gather_cluster_resources`.